### PR TITLE
scatter -> hist2d in check_correlation, add draw_correlation_heatmap

### PR DESCRIPTION
**Pull request by onaga1958**

### CODE REVIEW

File: modelgym/guru.py [hunk 1]
```diff
@@ -155,6 +156,18 @@ def check_class_disbalance(self, y):
 
         return candidates
 
+    def draw_correlation_heatmap(self, X, feature_indexes=None):
+        if feature_indexes is None:
```
Comment (anaderi) [inline modelgym/guru.py]: Why feature index instead of feature column(s)?

File: setup.py [hunk 2]
```diff
@@ -13,5 +13,5 @@
       author='The modelgym contributors',
       packages=["modelgym"],
       install_requires=["numpy", "scipy", "scikit-learn>=0.18",
-                        "matplotlib", "pandas"]
+                        "matplotlib", "pandas", "seaborn"]
```
Comment (anaderi) [inline setup.py]: You should also add the module to conda dependencies at environment folder.

File: modelgym/guru.py (hunk 1 above)
Comment (onaga1958) [reply modelgym/guru.py]: To make subscriptions on graphs and to release a user from writing _get_feature by himself. I think, it's useful in case of list of list.

Comment (anaderi) [reply modelgym/guru.py]: keeping track of column indexes is difficult since those can change during array transformation.. let's treat 'feature_indexes' either as list of integers or list of strings in which case you should resolve it to index inside the method.

File: environment/environment.yaml [hunk 3]
```diff
@@ -11,6 +11,7 @@ dependencies:
   - networkx=1.11
   - xgboost=0.6a2
   - lightgbm=2.0.4
+  - seaborn=0.8.1
```
Comment (anaderi) [inline environment/environment.yaml]: btw, could you please check that all required modules are also present at setup.py? (everything that is not related to classifiers, e.g. pyyaml)

Comment (anaderi) [reply environment/environment.yaml]: one more thing 
- make colormap the same for different plots, and 
- allow user to pass params to plotting functions like a) colormap, b) number of bins, c) fig_size
